### PR TITLE
[prometheus-logstash] add logstash exporter

### PR DESCRIPTION
**Pull request by ebuildy**
Signed-off-by: Thomas Decaux <ebuildy@gmail.com>

#### What this PR does / why we need it:

An exporter for logstash

#### Checklist

- [X] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [X] Chart Version bumped
- [X] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/prometheus-logstash-exporter/README.md
Comment (monotek) [inline charts/prometheus-logstash-exporter/README.md:30]: helm 2 is deprecated. i would not mention it any longer.

File: charts/prometheus-logstash-exporter/Chart.yaml
Comment (monotek) [inline charts/prometheus-logstash-exporter/Chart.yaml:4]: why 0.1.1 and not 0.1.0 as semver suggest for initial version?
see: https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase

File: charts/prometheus-logstash-exporter/README.md
Comment (monotek) [inline charts/prometheus-logstash-exporter/README.md:45]: helm 2 is deprecated. i would not mention it any longer.

File: charts/prometheus-logstash-exporter/README.md
Comment (monotek) [inline charts/prometheus-logstash-exporter/README.md:56]: helm 2 is deprecated. i would not mention it any longer.

File: charts/prometheus-logstash-exporter/README.md
Comment (monotek) [inline charts/prometheus-logstash-exporter/README.md:68]: helm 2 is deprecated. i would not mention it any longer.

File: charts/prometheus-logstash-exporter/ci/default-values.yaml
Comment (monotek) [inline charts/prometheus-logstash-exporter/ci/default-values.yaml:1]: imho the file is not needed if empty.
at least i thought the default values are always checked?

File: charts/prometheus-logstash-exporter/templates/deployment.yaml
Comment (monotek) [inline charts/prometheus-logstash-exporter/templates/deployment.yaml:6]: please use the more recent labeling defaults which are using templating.

you can copy it from the chart created by:

```
helm create prometheus-logstash-exporter
```

File: charts/prometheus-logstash-exporter/values.yaml
Comment (monotek) [inline charts/prometheus-logstash-exporter/values.yaml:126]: any reason this is disabled?
imho should be enabled as long as the defined values work out of the box.

File: charts/prometheus-logstash-exporter/Chart.yaml
Comment (monotek) [inline charts/prometheus-logstash-exporter/Chart.yaml:1]: helm 2 is deprecated so use v2 here.

**Review by monotek — CHANGES_REQUESTED** [review]

File: charts/prometheus-logstash-exporter/ci/default-values.yaml
Comment (ebuildy) [reply charts/prometheus-logstash-exporter/ci/default-values.yaml:1]: right, I copy it from elasticsearch chart, will remove it

File: charts/prometheus-logstash-exporter/values.yaml
Comment (ebuildy) [reply charts/prometheus-logstash-exporter/values.yaml:126]: copied from elasticsearch-exporter chart, I am changing it

**Comment by monotek** [comment]
@ebuildy  
Could you fix the linter errors please?

**Review by monotek — CHANGES_REQUESTED** [review]
@ebuildy please fix linter errors.

**Comment by TsofiaTsuriel1234** [comment]
Is this PR is ready for merge? when are you going to merge it?
Thanks!

**Comment by monotek** [comment]
No, it's not ready.

The DCO check fails: https://github.com/prometheus-community/helm-charts/pull/628/checks?check_run_id=1904680196
Pleas fix.

@prometheus-community/helm-charts-admins 
The Codeowners check needs approval of one of the repo owners: https://github.com/prometheus-community/helm-charts/pull/628/checks?check_run_id=1904680517

**Review by monotek — CHANGES_REQUESTED** [review]
@TsofiaTsuriel1234 
No, it's not ready.

The DCO check fails: https://github.com/prometheus-community/helm-charts/pull/628/checks?check_run_id=1904680196

@ebuildy  please fix.

@prometheus-community/helm-charts-admins
The Codeowners check needs approval of one of the repo owners: https://github.com/prometheus-community/helm-charts/pull/628/checks?check_run_id=1904680517

**Comment by oded-dd** [comment]
@ebuildy Any update on that? Great!

**Comment by pwlawe** [comment]
@monotek We have a need for this, I'd like to revive this work.  I'll make a new PR and reference this one.  FYI @ebuildy

**Comment by monotek** [comment]
Could you please do this?
https://github.com/prometheus-community/helm-charts/pull/628/checks?check_run_id=3666839649

**Pull request closed without merging**
